wdio/specs/homepage.js

```diff
@@ -117,4 +117,25 @@ describe('Municipality homepage', () => {
     const target = browser.getAttribute('.item.action.homepage-teaser:nth-child(4) .content .header a', 'target');
     assert(target == '_blank');
   });
+
+  it('should show QAs only for the chosen user type "residents" and the hebrew language', () => {
+    browser.url('/municipality-1/node/1?user_type=residents&language=he');
+    browser.waitForVisible('div=כמה סייעות יש בגן ילדים?');
+    browser.waitForVisible('div=באילו שעות מפנים את הזבל?');
+  });
+
+  it('should not show QAs for "businesses" when "residents" are the chosen user type ', () => {
+    assert(!browser.isVisible('div= האם גני ילדים זכאים לפטור מארנונה?'));
```

Comment:
Also here not checking that nothing from `business` is showing, this test is essentially the same as the one above, no need for them both.